main.nf

```diff
@@ -363,6 +511,7 @@ process sentieon_qc {
 		set group, id, file("${id}_qc.json") into qc_cdm
 		set group, id, file("${id}_qc.json") into qc_melt
 		file("*.txt")
+		//path "*versions.yml"
```

Comment:
Should this one be commented here?

Reply:
dont't know. I'm starting to regret adding the stubs into this PR

Reply:
I agree, it is hard to see what are critical changes in the diff

Reply:
No you're right. If the stubs and shell produce a versions.yml file the this needs to be accessible. There are a few examples of this that needs to be checked.